apps/ensindexer/src/api/index.ts

```diff
@@ -22,12 +22,12 @@ app.use(
   }),
 );
 
-// use root to redirect to the ENSAdmin website with the current server URL as ensnode parameter
+// use root to redirect to the ENSAdmin website with the current server URL as ensindexer parameter
 app.use("/", async (ctx) =>
-  ctx.redirect(`https://admin.ensnode.io/about?ensnode=${ensNodePublicUrl()}`),
+  ctx.redirect(`https://admin.ensnode.io/about?ensindexer=${ensIndexerPublicUrl()}`),
```

Comment:
this is an example where i think this variable should be `ensnode=${ensNodeApiUrl()}` or something like that

Reply:
Sure, it makes sense to keep `ensnode` for client requests. Especially that the ENSNode SDK is coming with:
- #317